pimd/pim_mroute.c

```diff
@@ -360,10 +360,9 @@ static int pim_mroute_msg_wrongvif(int fd, struct interface *ifp,
 	*/
 
 	if (!ifp) {
-		if (PIM_DEBUG_MROUTE)
-			zlog_debug(
-				"%s: WRONGVIF (S,G)=%s could not find input interface for input_vif_index=%d",
-				__func__, pim_str_sg_dump(&sg), msg->im_vif);
+		zlog_warn(
```

Comment:
What do we expect the operator to do here with this warning?  additionally any fast flowing stream will immediately fill the log with this message as it comes in on the wrong interface.  Seems like a bad idea to me.